ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-upgrade.yaml

```diff
@@ -148,6 +148,7 @@ objects:
           touch /tmp/shared/exit;
         fi; 
         cp -r /go/src/github.com/openshift/openshift-azure/_data /tmp/shared &>/dev/null
+        cp /go/src/github.com/openshift/openshift-azure/test/overrides /tmp/shared
```

Comment:
The overrides apply only in the conformance template. Please remove from elsewhere.